docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>What's approximate about this? Sounds pretty complete to me? (For 1-D data at least)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>It's approximate because it's a convex objective and here we are stopping the algorithm "randomly". Given that you have bounds on the convergence, you should monitor those and perform backtracking linesearch or similar to choose the gradient step sizes, and stop at a given precision. But it's nitpicking agreed, this is just to say that we don't consider this as a proper full fledged solution for linear regression.</p>